demo/index.html

```diff
@@ -25,6 +26,11 @@
         <button class="js-dialog-btn-close">Close dialog 1</button>
     </div>
 
+    <div id="my-dialog2" class="dialog js-dialog">
```

Comment:
Oh yeah for sure adding it for the consumer is the way to go i.e. being dynamic. Sorry I totally forgot we were doing it that way. So yeah ignore what I said as we shouldn't be hard coding it in.